back/spec/fixtures/locales/en.yml

```diff
@@ -185,7 +185,8 @@ en:
       latitude: 'Latitude'
       likes_count: 'Likes'
       location: 'Location'
-      longitude: 'Longitude' 
+      longitude: 'Longitude'
+      neutral_reactions_count: 'Unsure'
```

Comment:
I get that these are fixtures, so don't really matter, but I think this meaning is not good, so maybe something to check with proofreading/translations.

'Unsure' is the opposite of 'sure', but I think what we want here is actually 'neutral' == no strong opinion (not a 'like' or 'dislike', not an 'upvote' or 'downvote', etc.